zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -500,6 +501,10 @@ public void loadData() throws IOException, InterruptedException {
         takeSnapshot();
     }
 
+    public boolean compareDigest(TxnHeader header, Record txn, TxnDigest digest) {
```

Comment:
Is this method needed? It's never actually called anywhere. If it's not needed, we should remove it.